flink-libraries/flink-state-processing-api/src/test/java/org/apache/flink/state/api/output/KeyedStateBootstrapOperatorTest.java

```diff
@@ -43,11 +48,53 @@
  */
 public class KeyedStateBootstrapOperatorTest {
 
-	private static final ValueStateDescriptor<Long> descriptor = new ValueStateDescriptor<Long>("state", Types.LONG);
+	private static final ValueStateDescriptor<Long> descriptor = new ValueStateDescriptor<>("state", Types.LONG);
+
+	private static final Long EVENT_TIMER = Long.MAX_VALUE - 1;
+
+	private static final Long PROC_TIMER = Long.MAX_VALUE - 2;
 
 	@Rule
 	public TemporaryFolder folder = new TemporaryFolder();
 
+	@Test
+	public void testTimerStateRestorable() throws Exception {
+		Path path = new Path(folder.newFolder().toURI());
+
+		OperatorSubtaskState state;
+		KeyedStateBootstrapOperator<Long, Long> bootstrapOperator = new KeyedStateBootstrapOperator<>(0L, path, new TimerBootstrapFunction());
+		try (KeyedOneInputStreamOperatorTestHarness<Long, Long, TaggedOperatorSubtaskState> harness = getHarness(bootstrapOperator)) {
```

Comment:
I know the other existing tests are doing this as well, but:

for the sake of readability, could be nice to put this in a separate method, called say `processElements(Harness<IN>, IN... )`.

Reply:
👍